controllers/community.server.controller.ts

```diff
@@ -80,30 +58,18 @@ export async function communityCreate(
 ) {
   const authToken = req.get(config.get('authToken'));
 
-  // Set community fields to an object for passing to the model
-  const communityParams: IValidation<CreateCommunityDTO> = {
-    name: {
-      value: req.body.name,
-      valid: req.body.name.length > 0,
-    },
-    img: {
-      value: req.body.img,
-      valid: true,
-    },
-    description: {
-      value: req.body.description,
-      valid: req.body.description != undefined,
-    },
-  };
+  const schema = Joi.object<CreateCommunityDTO>({
+    name: string().min(3).required(),
+    description: string().allow(['']).required(),
+    img: array().items(string().uri()).max(3).required(),
```

Comment:
CreateCommunityDTO set img as string type, but here for img we are checking if it is an array?

Reply:
good catch. Have fixed 👍🏻